packages/itwinui-variables/src/themes/light-hc.scss

```diff
@@ -4,66 +4,159 @@
 @use '../utils/hex' as *;
 @use '../utils/get-hsl' as *;
 
-$base-colors: (
-  // Background colors - default
-  background-1: hsl(200 12% 100%),
-  background-2: hsl(200 12% 98%),
-  background-3: hsl(200 12% 94%),
-  background-4: hsl(200 12% 88%),
-  background-5: hsl(200 12% 80%),
-  background-1-hover: hsl(0 0% 95%),
-  //
-  // Background colors - statuses subtle
-  background-subtle-primary: hsl(200 25% 95%),
-  background-subtle-positive: hsl(135 15% 95%),
-  background-subtle-warning: hsl(48 20% 95%),
-  background-subtle-negative: hsl(0 36% 96%),
-  //
-  // Foreground colors - default
-  foreground-1: hsl(0 0% 0%),
-  foreground-2: hsl(0 0% 14%),
-  foreground-3: hsl(0 0% 28%),
-  foreground-4: hsl(0 0% 33%),
-  foreground-5: hsl(0 0% 70%),
-  //
-  // Foreground colors - statuses
-  foreground-primary: hsl(200 100% 23%),
-  foreground-positive: hsl(140 100% 16%),
-  foreground-warning: hsl(45 100% 19%),
-  foreground-negative: hsl(0 100% 28%),
-  foreground-primary-hover: hsl(203 100% 34%),
-  foreground-positive-hover: hsl(95 72% 27%),
-  foreground-warning-hover: hsl(33 89% 41%),
-  foreground-negative-hover: hsl(0 91% 33%)
-);
+// #region SASS variables
+$bg-hue: 203;
+$bg-saturation: 5%; // Less saturated than base theme.
+$bg-lightness-gap: 4%;
+$fg-lightness-gap: 6%;
+// #endregion
+
+// #region Background colors
+// Lightest @ top, darkest @ bottom
+$bg-0: hsl($bg-hue $bg-saturation 100%);
+$bg-1: hsl($bg-hue $bg-saturation (100% - $bg-lightness-gap * 0.5));
```

Comment:
Same as other comment - have you thought about moving this color to be outside of the series? Maybe call it `$bg-0b` instead of `$bg-1`?